lib/main.coffee

```diff
@@ -356,6 +358,15 @@ module.exports = Hydrogen =
                     @watchSidebar.show()
         @watchKernelPicker.toggle()
 
+    showSessionPicker: ->
```

Comment:
Hydrogen doesn't really need to know about sessions.

We want users to select kernels (not just sessions).
Rather than a `showSessionPicker`, we need a `showWSKernelPicker`.
This change will help get rid of `gatewayKernelFromSession`.